src/components/fields/ArrayField.js

```diff
@@ -93,6 +100,78 @@ class ArrayField extends Component {
     const {items} = this.state;
     const {fields, definitions} = this.props.registry;
     const {SchemaField} = fields;
+    if (isFixedItems(schema)) {
```

Comment:
I also realize that FixedArrayField and normal ArrayField has many lines in common. I think I'm going to split `render()` into several stateless components, `ArrayFieldItem` `NormalArrayField` `FixedArrayField` etc.